source/BoardingPanel.cpp

```diff
@@ -305,9 +305,7 @@ bool BoardingPanel::KeyDown(SDL_Keycode key, Uint16 mod, const Command &command,
 		// to your ship in peace. That is to allow the player to "cancel" if
 		// they did not really mean to try to capture the ship.
 		bool youAttack = (key == 'a' && (yourStartCrew > 1 || !victim->RequiredCrew()));
-		bool enemyAttacks = defenseOdds.Odds(enemyStartCrew, yourStartCrew) > .5;
-		if(isFirstCaptureAction && !youAttack)
-			enemyAttacks = false;
+		bool enemyAttacks = !isFirstCaptureAction && !youAttack && defenseOdds.Odds(enemyStartCrew, yourStartCrew) > .5;
```

Comment:
```suggestion
		bool enemyAttacks = !isFirstCaptureAction && defenseOdds.Odds(enemyStartCrew, yourStartCrew) > .5;
```
If you choose to attack it doesn't matter what the enemy chooses.